src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -4060,7 +4060,7 @@ public void ParseErrorLog()
             const string InvalidSarifVersion = @"C:\MyFolder\MyBinary.xml,version=42";
             parsedArgs = DefaultParse(new[] { $"/errorlog:{InvalidSarifVersion}", "a.cs" }, baseDirectory);
             parsedArgs.Errors.Verify(
-                // error CS2046: Command-line syntax error: 'C:\MyFolder\MyBinary.xml,version=42' is not a valid value for the '/errorlog:' option. The value must be of the form '<file>[,version={1|1.0|1.0.0|2|2.1|2.1.0}]'.
+                // error CS2046: Command-line syntax error: 'C:\MyFolder\MyBinary.xml,version=42' is not a valid value for the '/errorlog:' option. The value must be of the form '<file>[,version={1|1.0|2|2.1}]'.
```

Comment:
>// error CS2046: Command-line syntax error: 'C:\MyFolder\MyBinary.xml,version=42' is not a valid value for the '/errorlog:' option. The value must be of the form '<file>[,version={1|1.0|2|2.1}]'. [](start = 16, length = 195)

Comments do not verify anything. Please add tests that demonstrate that the options that you removed from the list are indeed errors. Please also add tests for VB